packages/govtnz-ds-website/src/components/layout.tsx

```diff
@@ -26,6 +26,10 @@ const Layout = (props: Props) => {
       document.documentElement.classList.add('theme-default', 'js');
     }
 
+    if (typeof window !== 'undefined') {
```

Comment:
Why are we adding the `js` class twice, on line 26 and line 30?

Reply:
removed, we don't need it here.

Reply:
Thanks.. the preceding code checks for whether `document.body.classList` exists before using it but this code doesn't. Perhaps we should move this code into that `if` block on line 26?